src/spartan_http_handler.erl

```diff
@@ -0,0 +1,163 @@
+-module(spartan_http_handler).
+
+-export([
+    init/3,
+    rest_init/2,
+    allowed_methods/2,
+    content_types_provided/2,
+    process/2
+]).
+
+-include_lib("dns/include/dns.hrl").
+-include_lib("erldns/include/erldns.hrl").
+
+init(_Transport, Req, Opts) ->
+    {upgrade, protocol, cowboy_rest, Req, Opts}.
+
+rest_init(Req, Opts) ->
+    {ok, Req, Opts}.
+
+allowed_methods(Req, State) ->
+    {[<<"GET">>], Req, State}.
+
+content_types_provided(Req, State) ->
+    {[
+        {{<<"application">>, <<"json">>, []}, process}
+    ], Req, State}.
+
+process(Req, State = [version]) ->
+    Applications = application:which_applications(),
+    {spartan, _V, Version} = lists:keyfind(spartan, 1, Applications),
+    Body =
+        jsx:encode([
```

Comment:
json clients don't expect ordering in objects so we don't need to worry about this.